### PR TITLE
Add details about redis data usage to contribution guide

### PR DESCRIPTION
**Pull request by farukaydin**
When I work on https://github.com/rubygems/rubygems.org/issues/947, I realized that it'd be useful to have details about how we can use redis production data in development environment

### CODE REVIEW

**Comment by farukaydin** [comment]
Changed the text with small improvements to make it more understandable, updated branch.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -163,6 +163,10 @@ directory. The proper directory will be full of .gem files.
 #### Getting the data dumps
 * You can use rubygems.org data [dumps](https://rubygems.org/pages/data) to test
 application in development environment especially for performance related issues.
+To use redis data dumps, you should check redis configuration file to make sure
+configuration of the filename where to dump the db and it's directory point to the
+location of the redis dump file you have just downloaded from rubygems.org
+[(See more detail about redis data import)](http://-stackoverflow.com/questions/14497234/how-to-recover-redis-data-from-snapshotrdb-file-copied-from-another-machine)
```
Comment (JuanitoFatas) [inline CONTRIBUTING.md]: URL should be `http://stackoverflow.com/questions/14497234/how-to-recover-redis-data-from-snapshotrdb-file-copied-from-another-machine` :blush:

Comment (farukaydin) [inline CONTRIBUTING.md]: :smile: Thanks @JuanitoFatas I'll update it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -163,6 +163,10 @@ directory. The proper directory will be full of .gem files.
 #### Getting the data dumps
 * You can use rubygems.org data [dumps](https://rubygems.org/pages/data) to test
 application in development environment especially for performance related issues.
+To use redis data dumps, you should check redis configuration file to make sure
+configuration of the filename where to dump the db and it's directory point to the
+location of the redis dump file you have just downloaded from rubygems.org
+[(See more detail about redis data import)](http://stackoverflow.com/questions/14497234/how-to-recover-redis-data-from-snapshotrdb-file-copied-from-another-machine)
```
Comment (arthurnn) [inline CONTRIBUTING.md]: > you should check redis configuration file to make sure configuration of the filename where to dump the db...

this part is a bit confusing to me, honestly I didnt understand it.

Comment (farukaydin) [inline CONTRIBUTING.md]: @arthurnn I wrote it to emphasize this specific part of redis.conf file.
```shell
# The filename where to dump the DB
dbfilename mydump.rdb

# The working directory.
#
# The DB will be written inside this directory, with the filename specified
# above using the 'dbfilename' configuration directive.
#
# The Append Only File will also be created inside this directory.
#
# Note that you must specify a directory here, not a file name.
dir /usr/local/var/db/redis/
```
We can change this part like "You should check redis.conf file to make sure what should be the redis dump filename and where it should be"

**Comment by arthurnn** [comment]
Thanks a lot @farukaydin . I end-up merging #1080 , which had a more simple statment IMO.
Fell free to iterate on top of that, if you think it can get better.

thanks

**Pull request closed without merging**